load.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * SEO Module
+ *
+ * @package hm-platform/seo
+ */
+
+namespace HM\Platform;
```

Comment:
This should be `HM\Platform\SEO` no?

Reply:
I didn't think so, doesn't make much difference unless I was to move the module load function out.